### PR TITLE
Fix #975 : Reworks avatar sizes and spacing across breakpoints

### LINKED ISSUE #975: Photos overlap some text on Team/Authors/Spotlight pages
[Issue reported on Twitter](https://twitter.com/wpeditorial/status/1283830085380911110).

At small screen sizes the photo can overlap the person's name if it is long enough. We'll need to:

- [ ] Add a `tiny` photo size to the `$_person-photo` Sass map.
- [ ] Tweak the incorporating breakpoint declarations to incorporate it.
- [ ] Verify using a smartphone.

Also see: https://github.com/a11yproject/a11yproject.com/issues/948

### PR DESCRIPTION
**Pull request by beckkramer**
This is a fix for Issue #975 

**This PR:**

- Adds a height/width Sass mixin for different photo sizes to pare down on code
- Adds a `tiny` image size and adjusts the other values as some weren't being used
- Adjusts the spacing and orientation of the avatar images per breakpoint

**Tasks from the issue:**

- [x] Add a tiny photo size to the $_person-photo Sass map.
- [x] Tweak the incorporating breakpoint declarations to incorporate it.
- [x] Verify using a smartphone.

**Screenshots of the element at different breakpoints:**

Screenshot_12
Screenshot_16
Screenshot_15
Screenshot_14
Screenshot_13

### CODE REVIEW

File: src/css/components/_c-person.scss [hunk 1]
```diff
@@ -18,53 +19,60 @@ $_person-photo: (
 	}
 }
 
+@mixin photo($size: tiny) {
```
Comment (ericwbailey) [inline src/css/components/_c-person.scss:22]: Could you update this to add `_person-` to the mixin, and then move it up to after `$_person-photo`? The system for organizing it this way is:

- All Sass logic applicable to only this partial declared first, followed by the partial's class declarations.
- Namespace Sass logic applicable to only this partial with `_` and `person-`
    - `_` indicates that's its a partial relevant to this partial only, and not "global" logic that lives in `[logic/](https://github.com/a11yproject/a11yproject.com/tree/main/src/css/logic)`.
    - `person-` indicates it's for the `c-person` component, which can be helpful for searching through the repo's code.

File: src/css/components/_c-person.scss [hunk 2]
```diff
@@ -18,53 +19,60 @@ $_person-photo: (
...
 // Photos use background images since we can't control the dimensions of
 // submitted avatars. This centers whatever is added, which hopefully creates a
 // good cropping effect.
 .c-person__photo {
+	@include photo(tiny);
+
 	background-position: center;
 	background-size: contain;
-	border-radius: map-get($_person-photo, small) * 2;
+	border-radius: 100%;
```
Comment (ericwbailey) [inline src/css/components/_c-person.scss:36]: Clever!

File: src/css/components/_c-person.scss [hunk 3]
```diff
@@ -18,53 +19,60 @@ $_person-photo: (
...
 	cursor: pointer;
 	display: inline-block;
-	height: map-get($_person-photo, small);
-	margin-top: -3.5rem;
+	margin-top: -2.75rem;
 	position: absolute;
-		right: 0.25rem;
-	width: map-get($_person-photo, small);
+		right: calc(50% - #{map-get($_person-photo, tiny)/2});
```
Comment (ericwbailey) [inline src/css/components/_c-person.scss:41]: This is also really nice 😀

File: src/css/components/_c-person.scss [hunk 4]
```diff
@@ -18,53 +19,60 @@ $_person-photo: (
...
 
 	// Breakpoints
 	@include mappy-bp(wrist-large) {
-		right: 6vw;
+		@include photo(small);
```
Comment (ericwbailey) [inline src/css/components/_c-person.scss:45]: Another style thing we're doing is using a blank line between mixin includes and declaration blocks. This all gets compiled down to minified CSS, so we don't mind the extra space here in the Sass.

Comment (beckkramer) [reply src/css/components/_c-person.scss]: I made an adjustment on this front, but let me know if I've put the space on the wrong side of things (if possible, could you use the "suggestion" feature to show me what you mean if I've misinterpreted it? Thank you!)

File: src/css/components/_c-person.scss (hunk 1 above)
Comment (beckkramer) [reply src/css/components/_c-person.scss]: I've made some tweaks here, thanks for the info about the preferred styles!

**Comment by ericwbailey** [comment]
@beckkramer Looks great! Merge away!

**Comment by beckkramer** [comment]
Will do, thanks for the review!

**Comment by beckkramer** [comment]
Actually, spoke too soon. I don't have write access, so someone else will have to merge when they're able @ericwbailey

**Comment by ericwbailey** [comment]
Merged. Thank you so much for doing this 🎉

**Pull request merged**